apps/src/templates/studioHomepages/CourseBlocks.jsx

```diff
@@ -162,33 +162,44 @@ class CourseBlocksCsfLegacy extends Component {
   }
 }
 
-export class CourseBlocksHoc extends Component {
+export class CourseBlocks extends Component {
   static propTypes = {
-    isInternational: PropTypes.bool
+    tiles: PropTypes.arrayOf(PropTypes.string).isRequired
   };
 
   componentDidMount() {
-    const tiles = this.props.isInternational
-      ? ['#dance-2019', '#aquatic', '#frozen', '#hourofcode']
-      : ['#dance-2019', '#aquatic', '#oceans', '#flappy'];
-
-    tiles.forEach((tile, index) => {
+    this.props.tiles.forEach((tile, index) => {
       $(tile).appendTo(ReactDOM.findDOMNode(this.refs[index]));
     });
   }
 
   render() {
     return (
       <div className="tutorial-row">
-        <ProtectedStatefulDiv className="tutorial-block" ref="0" />
-        <ProtectedStatefulDiv className="tutorial-block" ref="1" />
-        <ProtectedStatefulDiv className="tutorial-block" ref="2" />
-        <ProtectedStatefulDiv className="tutorial-block" ref="3" />
+        {this.props.tiles.map((tile, index) => (
+          <ProtectedStatefulDiv className="tutorial-block" ref={index} />
```

Comment:
Looks like you're using [string refs](https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs) which are now considered "legacy" (but not officially deprecated? :man_shrugging:)  Might be a good idea to use a [callback ref](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs) instead, just to be more future-proof.

Reply:
Done.